Example/App.js

```diff
@@ -37,11 +37,7 @@ const SCREENS = {
   },
 };
 
-class MainScreen extends React.Component {
-  static navigationOptions = {
-    title: '📱 React Native Screens Examples',
-  };
-  render() {
+const MainScreen = ({ navigation }) => {
     const data = Object.keys(SCREENS).map((key) => ({ key }));
```

Comment:
Looks like all the code down from here has one more tabulation too.

Reply:
hi @WoLewicki  i change your last suggestion but accidentally i commit everything like `yarn.lock` and `Podfile.lock`, but i fix everything in the last commit.